how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,59 @@
+# Kata Containers with NEMU
+Kata Containers relies by default on Qemu hypervisor in order to spawn the virtual machines running containers. [NEMU](https://github.com/intel/nemu) is a fork of Qemu focused on increasing the security through 3 different ways:
+- Reducing the number of lines of code.
+- Removing all legacy devices.
+- Reducing the emulation as far as possible.
+
+As the goal for Kata Containers is to increase the security, this is a logical step to get Kata relying on NEMU as its hypervisor.
+
+## How to get started
+
+This section will describe how to run Kata Containers with NEMU, first by explaining how to download, build and install it. Then this document will walk through the steps needed to update your Kata Containers configuration in order to run with NEMU. 
+
+### Pre-requisites
+This document requires Kata Containers to be installed in your system.
+
+### NEMU
+___Download, build and install___
+```bash
+$ git clone https://github.com/intel/nemu.git
+$ cd nemu
+$ git fetch origin/master
+$ git checkout experiment/automatic-removal
```

Comment:
It might be worth explaining why this step is required as it sounds potentially a bit "scary" (automatic removal of what exactly?!? :)